modules/ocf_desktop/manifests/xsession.pp

```diff
@@ -52,16 +52,9 @@
   }
 
   # wallpaper symlink
-  if $::lsbdistcodename == 'jessie' {
-    $wallpaper = $staff_only ? {
-      true  => 'background-staff.png',
-      false => 'background.png',
```

Comment:
Another change to make is to remove both these background images from this repo, since we don't use them any more.